packages/dx-react-chart-demos/src/demo-sources/bar-chart/rotated-bar.jsxt

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';<%&additionalImports%>
```

Comment:
Should we add a rotated line chart demo too?

Reply:
Why? It change only visual presentation.  And rotated bar chart is the most common use case. Other types of series are rarely used to rotate.